### ISSUE TITLE
Memory card directory not created automatically and gives unfriendly error

### ISSUE THREAD

**Issue by DannyJJK** [KieronJ/rpsx#3]
Version: rpsx #8af932d
OS: macOS Monterey 12.5 (21G72)
Hardware: MacBook Pro (13-inch, 2019)

When I initially ran the emulator I had this error:
```
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Os { code: 2, kind: NotFound, message: "No such file or directory" }', src/psx/peripherals/memory_card.rs:68:36
```

I decided to take a look at the code and found the directory it was looking for: `./cards` so I created this directory which stopped the error coming up.

If the user is expected to create this manually, a more friendly error message would be fine, or can this directory be created automatically by the emulator?

**Comment by KieronJ** [comment]
Yeah memory card I/O is fairly unfriendly at the moment. I can have it create the directory automatically which should be a lot more convenient.

There is also a timing bug I'm still trying to find which is affecting the ability to load and save many games.

**Comment by KieronJ** [comment]
This should be fixed by 5c0fe198832d8ae948038cccfc5da33d0246be8e.
Thanks for raising this issue.
